### DISCUSSION TITLE
Add docs so that devs know deploying to object rather than resource account is the latest best practice

### DISCUSSION THREAD

**Discussion by 0xbe1** [aptos-labs/aptos-developer-discussions#252] [Dev Doc and Tutorial Feedback]
It seems according to https://github.com/aptos-labs/aptos-developer-discussions/discussions/191#discussioncomment-9377463 we should deploy to object rather than resource account. But this is not told anywhere in aptos.dev and I find it a missing. Can you help update docs to include this, also let people know object is preferred over resource account?

---

UPDATE: just saw the exact content in the preview docs https://preview.aptos.dev/en/build/smart-contracts/learn-move/advanced-guides/object-code-deployment#deploy-code-to-an-object but in https://preview.aptos.dev/en/build/smart-contracts/learn-move/advanced-guides/resource-accounts it still suggests devs can deploy to resource account. should make it clear that object is preferred

**Comment by kshitijc1506** [comment]
Agreed! Will address this, thank you!
